docs/developers/build-starrocks/build_starrocks_on_ubuntu.md

```diff
@@ -0,0 +1,70 @@
+### Prerequisite
+```
+sudo apt-get install automake binutils-dev bison byacc ccache cmake flex libiberty-dev libtool maven zip -y
+```
+
+### Compiler
+Check the compilers(GCC, Python, JDK) are installed in the machine, if not install them
+```
+sudo apt-get install gcc g++ default-jdk python3 python-is-python3 -y
```

Comment:
Only relying on apt-get is not necessarily able to install the appropriate version of gcc, cmake or python and other tools.

Reply:
If download the binary failed, we can add it in the FAQ.